src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -330,10 +328,11 @@ export class InstantiateModuleTransformer extends ModuleTransformer {
 
       // then do export bindings of re-exported dependencies
       if (externalExportBindings) {
-        externalExportBindings.forEach((binding) => {
-          setterStatements.push(
-            parseStatement `$__export(${binding.exportName}, $__m.${binding.importName});`
-          );
+        externalExportBindings.forEach(({exportName, importName}) => {
+          let statement = importName === null ?
```

Comment:
This is kind of hacky. I might go back and revisit the InstantiateModuleTransformer to see how this can be done in a cleaner way.